share/nisar/defaults/gcov.yaml

```diff
@@ -570,14 +570,27 @@ runconfig:
                 # Default: True
                 qa_reports: true
 
+            software_config:
+
+                # True to use memory map(s) to cache select Dataset(s).
```

Comment:
```suggestion
                # True to use memory map(s) to cache select dataset(s).
```
Is there a specific reason "Dataset" is capitalized here? If not, we should lowercase it.

Reply:
For HDF5 Datasets, we've been using a capital "D".